### PR TITLE
Update data path for all rseqc modules at once

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by maxulysse** [comment]
replace:
https://github.com/nf-core/modules/pull/4862
https://github.com/nf-core/modules/pull/4863
https://github.com/nf-core/modules/pull/4864
https://github.com/nf-core/modules/pull/4865
https://github.com/nf-core/modules/pull/4867
https://github.com/nf-core/modules/pull/4876

File: modules/nf-core/rseqc/readdistribution/tests/main.nf.test [hunk 1]
```diff
@@ -13,11 +13,11 @@ nextflow_process {
         when {
             process {
                 """
-                input[0] = [ 
-                    [ id:'test', single_end: false ], // meta map
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
```
Comment (asp8200) [inline modules/nf-core/rseqc/readdistribution/tests/main.nf.test:17]: What happened to `single_end: false`?

File: modules/nf-core/rseqc/readdistribution/tests/main.nf.test.snap [hunk 2]
```diff
@@ -5,8 +5,7 @@
                 "0": [
                     [
                         {
-                            "id": "test",
-                            "single_end": false
+                            "id": "test"
```
Comment (asp8200) [inline modules/nf-core/rseqc/readdistribution/tests/main.nf.test.snap:8]: Losing that `"single_end": false` here.

File: modules/nf-core/rseqc/readduplication/tests/main.nf.test [hunk 3]
```diff
@@ -13,24 +13,23 @@ nextflow_process {
         when {
             process {
                 """
-                input[0] = [ [ id:'test', single_end: false ], // meta map
-                             file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
-                           ]
+                input[0] = Channel.of([
```
Comment (asp8200) [inline modules/nf-core/rseqc/readduplication/tests/main.nf.test:16]: losing `single_end: false ` here.

**Review by asp8200 — COMMENTED** [review]
You lost something. It might be okay though.

**Pull request merged**